src/ng_link/dispim_link.py

```diff
@@ -89,42 +94,17 @@ def generate_dispim_link(
         tile_paths[0]
     )
 
-    # Gather cross channel xml info, only care about transforms
-    # Massaging transforms into same format as 'intertile_transforms'.
-    channel_paths: dict[int, str] = xml_parsing.extract_tile_paths(
-        cross_channel_xml_path
+    channels: list[int] = link_utils.get_unique_channels_for_dataset(
```

Comment:
@jwong-nd Here is the function call I use to get just the list of unique channels in a dataset. It uses boto3 to look at the s3 path